modules/claims_api/app/controllers/concerns/claims_api/disability_compensation_validations.rb

```diff
@@ -530,5 +531,18 @@ def validate_treatment_center_names!
         treatment
       end
     end
+
+    def validate_form_526_direct_depost!
+      direct_deposit = form_attributes['directDeposit']
+      return if direct_deposit.blank?
+
+      bank_name = direct_deposit['bankName']
+      if bank_name.blank?
+        raise ::Common::Exceptions::InvalidFieldValue.new(
+          'directDeposit.bankName',
+          direct_deposit['bankName']
+        )
+      end
+    end
```

Comment:
In the schema if `directDeposit` is present then "accountType", "accountNumber", "routingNumber" are all required, so the check that the object is present is enough to verify we need `bankName` as well since it is kind of all or nothing.

Reply:
@rockwellwindsor-va When I send `"financialInstitutionName": ""`, I get a 202. Am I misunderstanding your comment above? thanks.
<img width="1008" alt="Screenshot 2024-11-04 at 2 52 20 PM" src="https://github.com/user-attachments/assets/ae03d038-9af9-4ff0-91d7-35966d91e5e8">

Reply:
@stiehlrod I think that looks like a v2 request is that right?  This is specifically an update for v1 and the field we are look at is `bankName` which is probably the v1 equivalent to `financialInstitutionName ` in v2.

Reply:
Although this might also show we need a ticket for v2, do you happen to know if that failed going to the docker container?

Reply:
@rockwellwindsor-va I tested in v1 and got the expected behavior. In v2, it "Successfully submitted to Docker container with response"